docs/develop/go/temporal-nexus.mdx

```diff
@@ -466,7 +472,7 @@ To install the latest version of the `tcld` CLI, run the following command:
 brew install temporalio/brew/tcld
 ```
 
-If you don’t already have certificates, you can generate them for mTLS Worker authentication using the command below:
+If you don't already have certificates, you can generate them for mTLS Worker authentication using the command below:
```

Comment:
Why are we changing the apostrophe style?

Reply:
The old style was incorrect from an earlier docs conversion and causes problems on non-mac platforms.

Reply:
it was causing source formatting issues on non-mac platforms.